snc.sh

```diff
@@ -72,6 +73,12 @@ if test -z ${OPENSHIFT_INSTALL-}; then
     OPENSHIFT_INSTALL=./openshift-baremetal-install
 fi
 
+if [[ ${CLUSTER_PATCH} == "enabled" ]]
```

Comment:
Please name more intuitively

Reply:
`USE_PATCHED_RELEASE_IMAGE` is more descriptive, but I'm not sure how to mention cert validity in there :)